automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala

```diff
@@ -112,6 +115,14 @@ trait GPAllocBeforeAndAfterAll extends GPAllocUtils with BeforeAndAfterAll {
         case Right(billingProject) =>
           IO(sys.props.put(gpallocProjectKey, billingProject.value)) >> createInitialRuntime(billingProject)
       }
+      bindAttempt <- IO.fromFuture(IO(ProxyRedirectClient.startServer)).attempt
```

Comment:
Start a local server for serving the redirect JS. This is done in the test suite `beforeAll`, so it only happens once.